Ginger/Ginger/Actions/ActBrowserElementEditPage.xaml.cs

```diff
@@ -73,7 +84,7 @@ private ePlatformType GetActionPlatform()
         private void SetVisibleControlsForAction()
         {
             ResetView();
-            ePlatformType ActivityPlatform = GetActionPlatform();
+            ePlatformType ActivityPlatform = GetActivityPlatform();
 
```

Comment:
you need to use act.Platform here instead of calcualting it again by calling GetActivityPlatform(), it won't work if Action is on Shared Repo!

Reply:
Above has been done.